code/style.css

```diff
@@ -1,7 +1,217 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+*{
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  background: #dce4ef;
+  font-family: 'Jost';
+}
+
+/* defines the behaviour of the navigqtionbar*/
+.header{
+  width: 100%;
+  height: 100px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 10px 60px 10px 20px;
+}
+
+nav a {
+  display: none;
+}
+
+nav {
+  color:#3c3c3b;
+  text-transform: uppercase;
+  display: inline-block;
+  padding: 15px;
+
+}
+
+nav a:hover {
+  color: #3c3c3b;
+  text-decoration: underline;
+  font-weight: bolder;
+}
+
+.logo {
+  height: 50px;
+}
+
+/*places the hero image in the background and centers it*/
+.hero-image {
+  background-image: url("nG_Header_gelb.png");
+  height: 250px;
+  background-position: center;
+  background-size: cover;
+  position: relative;
+}
+
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color:#274b8f;
+  padding-top: 75px;
+}
+
+/*definies the button on the whole page*/
+
+.button {
+  background-color: #f1c419;
+  border: none;
+  color: #3c3c3b;
+  padding: 15px 32px;
+  margin-top: 30px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  border-radius: 8px;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2), 0 3px 10px 0 rgba(0,0,0,0.19);
+}
+
+.button:hover{
+  box-shadow: 0 6px 8px 0 rgba(0,0,0,0.24), 0 8px 25px 0 rgba(0,0,0,0.19);
+}
+
+/* the design in the main body is done with grid*/
+.mainbody{
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+/*the next two sections defines the boxes for the form as well as the accordion*/
+form {
+  background-color:#83a6ce;
+  padding: 15px;
+  margin: 15px;
+  height: fit-content;
+  min-height: 400px;
+}
+
+section{
+  background-color: #83a6ce;
+  padding: 15px;
+  margin: 15px;
+  height: fit-content;
+  min-height: 400px;
+}
+
+/*defines the behaviour of the signup form*/
+
+label {
+  font-size: 16px;
+  font-family: sans-serif;
+  color: #3c3c3b;
+}
+
+input {
+  display: block;
+  width: 250px;
+  height: 30px;
+  font-size: 14px;
+  border: 0;
+  border-bottom: 2px solid;
+  border-radius: 5px;
+  border-color: #274b8f;
+  cursor: auto;
+  margin-bottom: 10px;
+}
+
+input:valid {
+  border-color:green;
+}
+
+.radiobutton-row{
+  display: flex;
+  justify-content: space-around;
+  max-width: 350px;
+  margin-left: 75px;
+}
+ /*defines the behaviour of the accordion*/
```

Comment:
I really like how you have explained through-out your code what you are doing and why. Really well done